src/app/pages/flex-page/blocks/RichTextBlock.scss

```diff
@@ -23,7 +23,7 @@
   > *:first-child {
     margin-top: 0;
   }
-  > *:last-child {
+  > *:not(h1, h2, h3, h4, h5, h6):last-child {
```

Comment:
not certain this is the best way to handle this, but without the margin the text descenders are cut off

Reply:
This appears to be a `line-height` issue in the pattern library. hero-h1 line-height is `5rem`. It should be `normal` rather than being specified. This dates back to ancient times when the pattern-library was first spec'd.